README.md

```diff
@@ -25,3 +25,22 @@ go run main.go -port 3000 -jwt "<jwt_token>"
 ➜ curl -XGET http://127.0.0.1:3000/pin\?filePath\=not_exist_file
 {"error": "open not_exist_file: no such file or directory"}
 ```
+
+there's a timeout of 60s, on timeout the error message returned is:
```

Comment:
upload of 6MB files happens in less than 10s typically. 
Downloads succeed immediately if it's present in the local ipfs node (which will be the case most of the time). If that's not the case, it ended up taking 10 minutes in my experiments for some cids.
So i've increased the download timeout to 12 minutes and added comments.
It would probably be something we want to add telemetry around in rudder.